### ISSUE TITLE
Not allowed to load local resource: file:///storage/emulated/0/Android/data/etc......

### ISSUE THREAD

**Issue by uptownjimmy** [chukwu/ionic3-Image-Cache#10]
Hello: not sure how to resolve this. I know it's a common problem in general, not specific to Ionic or to this plugin, but I wondered if anyone else had seen this and resolved it.  

I am testing on an Android device.

**Comment by aditbharadwaj** [comment]
yep iam having the same issue on android device it works good on ionic serve but on android it just say undefined and in console the above error

**Comment by itlpps** [comment]
i received this error when i run with livereload. 
`ionic cordova run android` works fine.

**Comment by CompRider** [comment]
Any success?

**Comment by aditbharadwaj** [comment]
> Any success?

I moved to a different plugin all togther

**Comment by aditbharadwaj** [comment]
> Any success?
i moved to different plugin link below:-
[this](https://github.com/zyra/ionic-image-loader)
